.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -2,24 +2,37 @@
 name: Bug Report
 about: Create a report to help us improve
 title: ''
-labels: Support
+labels: "Support, needs-triage"
 
 ---
 
 <!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!-->
 <!-- 请使用英文描述问题 -->
+<!-- Note, lacking of information will delay the handling of issue
+     See our github issue handling flow here:
+     https://github.com/emqx/emqx/blob/master/.github/ISSUE_TEMPLATE/asserts/issue-handling.png
+-->
 
-**Environment**:
+### Subject of the issue
+Describe your issue here.
 
+### Your environment
 - EMQ X version (e.g. `emqx_ctl status`): 
```

Comment:
For EMQ X 4.3 or newer, please provide the log archive generated by [node_dump tool](https://github.com/emqx/emqx/blob/master/bin/node_dump) 

Otherwise please provide below info: